(node:3791169) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-session-44491',
  containerIp: '10.0.3.173',
  mitmPort: 44491,
  autoAccessFixed: 1
}
2024-11-13 12:54:15.657 - [Info] MITM Version: 2
2024-11-13 12:54:15.660 - [Info] Auto Access Enabled: true
2024-11-13 12:54:15.660 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-13 12:54:15.671 - [Info] SSH man-in-the-middle server for 10.0.3.173 listening on 10.0.3.1:44491
2024-11-13 12:55:35.453 - [Debug] [Connection] Attacker connected: 218.92.0.212 | Client Identification: SSH-2.0-PuTTY
2024-11-13 12:55:37.263 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'none'
2024-11-13 12:55:37.264 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-13 12:55:37.680 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'password'
2024-11-13 12:55:37.681 - [Debug] [Auto Access] Attacker: 218.92.0.212, Threshold: 1, Attempts: 1
2024-11-13 12:55:37.682 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2024-11-13 12:55:37.683 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 1 attempts. Remaining: 5 attempts
2024-11-13 12:55:38.095 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'password'
2024-11-13 12:55:38.096 - [Debug] [Auto Access] Compromising the honeypot
2024-11-13 12:55:38.096 - [Debug] [Auto Access] Adding the following credentials: 'root:worinima'
2024-11-13 12:55:38.301 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-13 12:55:38.303 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-13 12:55:38.303 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.173
2024-11-13 12:55:41.854 - [Debug] [LXC] Authentication Failed
2024-11-13 12:55:41.854 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-13 12:55:41.855 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 2 attempts. Remaining: 4 attempts
2024-11-13 12:55:41.855 - [Debug] [Auth] Attacker authentication error: Error: All configured authentication methods failed
2024-11-13 12:55:41.855 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 3 attempts. Remaining: 3 attempts
2024-11-13 12:55:41.858 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-13 12:55:41.860 - [Debug] [LXC] Container's OpenSSH server closed connection
2024-11-13 12:55:43.314 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'password'
2024-11-13 12:55:43.314 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-13 12:55:43.315 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 4 attempts. Remaining: 2 attempts
2024-11-13 12:55:44.727 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'password'
2024-11-13 12:55:44.727 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-13 12:55:44.728 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 5 attempts. Remaining: 1 attempts
2024-11-13 12:55:46.140 - [Debug] [Auth] Attacker 218.92.0.212 trying to authenticate with 'password'
2024-11-13 12:55:46.141 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-13 12:55:46.145 - [Debug] [Auth] Attacker: 218.92.0.212 has so far made 6 attempts. Remaining: 0 attempts
2024-11-13 12:55:46.146 - [Debug] [Connection] Max Login Attempts Reached - Closing connection on attacker
2024-11-13 12:55:46.147 - [Debug] [Connection] Attacker closed the connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
